src/plugins/plot/MctPlot.vue

```diff
@@ -395,13 +395,14 @@ export default {
     this.openmct.objectViews.on('clearData', this.clearData);
     this.$on('loadingComplete', this.loadAnnotations);
     this.openmct.selection.on('change', this.updateSelection);
-    this.setTimeContext();
-
     this.yAxisListWithRange = [this.config.yAxis, ...this.config.additionalYAxes];
 
-    this.loaded = true;
+    this.$nextTick(() => {
```

Comment:
Interesting, why was this needed to make Vue 3 work?

Reply:
Eventually this ends up interacting with a $ref, so need `nextTick()` here to ensure it's available